snmp/datadog_checks/snmp/data/profiles/_generic-host-resources.yaml

```diff
@@ -65,3 +65,7 @@ metrics:
         column:
           OID: 1.3.6.1.2.1.25.3.3.1.1
           name: hrProcessorFrwID
+      - tag: deviceindex
```

Comment:
Could test this new tag by adding an entry to this fixture file: https://github.com/DataDog/integrations-core/blob/master/snmp/tests/compose/data/generic_host.snmprec

For example this following lines at the end of the file:
```
1.3.6.1.2.1.25.3.3.1.3.5|2|5
1.3.6.1.2.1.25.3.3.1.3.10|2|10
```

1.3.6.1.2.1.25.3.3.1.3.5|2|5 => hrDeviceIndex has value 5 at index 5
1.3.6.1.2.1.25.3.3.1.3.10|2|10 => hrDeviceIndex has value 10 at index 10

And adding a tag here:

https://github.com/DataDog/integrations-core/blob/ad957ad0a831e844bba884bf6aabe3536ee47563/snmp/tests/test_profiles.py#L1720-L1726

For example, something like this:
```python
    processor_tags = [
        ('1.3.6.1.3.81.16', 5),
        ('1.3.6.1.3.95.73.140.186.121.144.199', 10),
    ]
    for proc, deviceid in processor_tags:
        tags = common_tags + ['processorid:{}'.format(proc), 'deviceid:{}'.format(deviceid)]
        aggregator.assert_metric('snmp.hrProcessorLoad', count=1, tags=tags)
```